db/fixtures/miq_policy_sets.yml

```diff
@@ -105,7 +105,24 @@
           description: Initiate SmartState Analysis for Container Image
           action_type: default
           options: {}
-      Condition: []
+      Condition:
+        - name: do not scan image-inspectors image
```

Comment:
Not much difference, whatever you prefer.  Both wordings are better English than "image-inspectors".  (Sorry I'm nitpicking, it will be visible in UI.)
If you pick a wording with ', I believe quoting the whole string is valid YAML:

          description: "Don't scan image-inspectors image"

(or rename in UI, re-export and see)